### DISCUSSION TITLE
Welcome to react-likert-scale Discussions!

### DISCUSSION THREAD

**Discussion by Craig-Creeger** [Craig-Creeger/react-likert-scale#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by niamhb0yle** [comment]
Nothing to say other than I used this for my dissertation project and it was great!
